host/sgx/windows/sgxquoteproviderloader.c

```diff
@@ -58,19 +44,31 @@ void oe_load_quote_provider()
                                "not found\n");
             }
 
-            provider.get_qe_identity_info =
-                (sgx_get_qe_identity_info_t)GetProcAddress(
-                    _handle, SGX_QL_GET_QE_IDENTITY_INFO_NAME);
-            provider.free_qe_identity_info =
-                (sgx_free_qe_identity_info_t)GetProcAddress(
-                    _handle, SGX_QL_FREE_QE_IDENTITY_INFO_NAME);
+            provider.get_sgx_quote_verification_collateral =
```

Comment:
Could you please confirm that the collaterals are getting cached locally by the DCAP client?

Reply:
Please follow up with Azure DCAP client team.